content/en/docs/reference/access-authn-authz/authentication.md

```diff
@@ -484,35 +546,58 @@ token would return:
     "user": {
       "username": "janedoe@example.com",
       "uid": "42",
-      "groups": [
-        "developers",
-        "qa"
-      ],
+      # optional group memberships
+      "groups": ["developers", "qa"],
+      # optional additional information provided by the authenticator.
+      # this should not contain confidential data, as it can be recorded in logs
+      # or API objects, and is made available to admission webhooks.
       "extra": {
         "extrafield1": [
           "extravalue1",
           "extravalue2"
         ]
       }
-    }
+    },
+    # optional list audience-aware token authenticators can return,
+    # containing the audiences from the `spec.audiences` list for which the provided token was valid.
+    # if this is omitted, the audience is considered to be the Kubernetes API server.
+    "audiences": ["https://myserver.example.com"]
   }
 }
 ```
+{{% /tab %}}
+{{< /tabs >}}
 
 An unsuccessful request would return:
 
-```json
+{{< tabs name="TokenReview_response_error" >}}
+{{% tab name="authentication.k8s.io/v1" %}}
+```yaml
+{
+  "apiVersion": "authentication.k8s.io/v1",
+  "kind": "TokenReview",
+  "status": {
+    "authenticated": false,
+    # optional details about why authentication failed
```

Comment:
“optional” here is a little unclear:
 - Does the API allow keys other than `authenticated` and `error`?
 - is `authenticated ` required whereas `error` is optional?

Reply:
tried to clarify